### DISCUSSION TITLE
ReVanced support for Google log-in?

### DISCUSSION THREAD

**Discussion by YesTrustMe** [ReVanced/revanced-discussions#903] [Question] [answered]
I have a family member who finds YT media on their phone and either subscribes or watches the media on their phone via the official YT app.

Does ReVanced support linking of YT history and subscriptions via cloud (natively) without importing/exporting from Google Takeout?

**Comment by reisxd** [comment, marked as answer]
Yes. ReVanced patches the stock YT APK.
